modules/caddyhttp/metrics_test.go

```diff
@@ -0,0 +1,55 @@
+package caddyhttp
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Can we drop this dependency? Trying to lighten our dev requirements. Also makes these tests inconsistent with the rest of our code base.

Reply:
It was already transitively included via another module so I figured it was OK, but I can drop it.